float64 left_speed   # cm/s
float64 right_speed  # cm/s
float64 duration     # seconds
